qrenderdoc/Code/qrenderdoc.cpp

```diff
@@ -291,6 +295,27 @@ int main(int argc, char *argv[])
               .arg(configFilename));
     }
 
+    int replayHostIndex = -1;
+    if(parser.isSet(replayhost))
+    {
+      QString replayHost = parser.value(replayhost);
+      for(int i = config.RemoteHosts.count() - 1; i >= 0; i--)
```

Comment:
Is there a reason for this loop to run in reverse order?

Reply:
I notice that you use reverse for-loop [here](https://github.com/baldurk/renderdoc/blob/v1.x/qrenderdoc/Code/Interface/PersistantConfig.cpp#L217) so I just followed that ;) So I wonder why you use reverse order in `PersistantConfig::AddAndroidHosts()`?

Reply:
In that loop I'm removing all android hosts temporarily so the ones that are still available can be added back later. Since I'm removing items, if the loop ran forward it could skip over some if it didn't account for some indices being counted twice so the simplest solution was to iterate backwards.

Reply:
I misunderstood the purpose of that function. Thanks for your explanation! Changed the for-loop to normal.